tests/integration/components/polaris-page-test.js

```diff
@@ -421,7 +420,7 @@ test('it handles breadcrumbs correctly', function(assert) {
 
   let breadcrumbLink = breadcrumbLinks[0];
   assert.ok(
-    breadcrumbLink.href.indexOf('/home/the-beginning/13/27') > -1,
+    breadcrumbLink.href.indexOf('/home/the-beginning') > -1,
```

Comment:
IIRC this was previously testing that the breadcrumb handles models correctly - is this a capability we've lost now?

If that's the case, presumably host apps will need to refactor to use actions to transition between routes? In which case this is a breaking change so we'll need to be sure to document it and look at cutting a major release instead of a minor one...

Reply:
^^ follow-up thought to this, check if the documentation needs to change anywhere as a result of this. We don't have a docs page for `polaris-breadcrumbs` at the moment - it's included in the [`polaris-page` docs](https://github.com/smile-io/ember-polaris/blob/master/docs/page.md), which will probably need updating too.

Reply:
Agreed that we should match the original component - this was one of the first things we added, and it was intended to be as ember-friendly in terms of links as possible 😉 But that's something we can deal with via actions now anyway.

I'm good with shipping a breaking change - better now than later when the user base has had a chance to grow...